### PR TITLE
Primary tenants model modify

### PR DESCRIPTION
**Pull request by HHjolany**
## Link to Click Up ticket

[https://app.clickup.com/t/1fu94fy](https://app.clickup.com/t/1fu94fy)

## Describe this PR

primary tenants model changed to singular entity

#### _Checklist_

- [x] person entity removed
- [x] primary Tenant entity changed, to have two field

### CODE REVIEW

**Review by AnnaGolosova — APPROVED** [review]

**Review by EdwardAnoghena — APPROVED** [review]

File: AccountsApi/V1/Gateways/DynamoDbGateway.cs [hunk 1]
```diff
@@ -85,15 +82,15 @@ public async Task<List<Account>> GetAllAsync(Guid targetId, AccountType accountT
...
         }
 
         public async Task UpdateAsync(Account account)
         {
             if (account == null)
-                throw new ArgumentNullException($"{nameof(account).ToString()} ModelStateExtension shouldn't be null");
-            account.LastUpdatedAt = DateTime.Now;
+                throw new ArgumentNullException($"{nameof(account).ToString()} shouldn't be null");
+            account.LastUpdatedAt = DateTime.UtcNow;
             await _dynamoDbContext.SaveAsync(account.ToDatabase()).ConfigureAwait(false);
```
Comment (sandipchandra) [inline AccountsApi/V1/Gateways/DynamoDbGateway.cs:94]: Are we updating all the whole account object for update operation? Or we have some business rule to block some properties update?

**Review by sandipchandra — COMMENTED** [review]
One question about the update operation, rest of this looks fine.

File: AccountsApi/V1/Gateways/DynamoDbGateway.cs (hunk 1 above)
Comment (HHjolany) [reply AccountsApi/V1/Gateways/DynamoDbGateway.cs:94]: not all / the input update model apply on the existence entity and then passed to the **saveasync**

Comment (sandipchandra) [reply AccountsApi/V1/Gateways/DynamoDbGateway.cs:94]: Looks all good to me.

**Review by sandipchandra — APPROVED** [review]
Approved

**Review by LBHSPreston — APPROVED** [review]

**Pull request merged**